plugin/output/file/file.go

```diff
@@ -206,7 +206,7 @@ func (p *Plugin) createNew() {
 	logger.Errorf("tsFileName in createNew=%s", p.tsFileName)
 	f := fmt.Sprintf("%s%s", p.targetDir, p.tsFileName)
 	pattern := fmt.Sprintf("%s*%s%s%s", p.targetDir, fileNameSeparator, p.fileName, p.fileExtension)
-	matches, err := filepath.Glob(pattern)
```

Comment:
maybe here, we can handle error?

Reply:
I'm not sure for now, that err must be thrown, so err skipped as it was.

Reply:
if we can't run glob, I think we should do a fatal

Reply:
Oh, I thought that we can just log and ignore it. Added log.Fatalf